app/javascript/src/hyacinth_v1/components/digital_objects/new/DigitalObjectNew.js

```diff
@@ -43,14 +43,14 @@ function DigitalObjectNew() {
 
           <Collapse in={digitalObjectType != null && digitalObjectType !== ''}>
             <div>
```

Comment:
Hmm. Is this `div` still necessary? Can `Row` be directly under `Collapse`?

Reply:
You're right.  Not sure why it was originally written that way, but I'll remove it.  Maybe it had a class at one point and it was removed.